tests/unit/test_models.py

```diff
@@ -143,5 +143,25 @@ def test_from_dict(self):
         )
 
 
+class TestAccountingPeriod(unittest.TestCase):
+    def test_str(self):
+        self.assertEqual(
```

Comment:
Thanks for writing these tests. That makes it easy to verify!

Reply:
Glad you like it! Would you like to update the readme with these changed instructions or should I do it here? I just feel like we will have unnecessary merge conflicts if I change it here.